cmake/CompilerOptions.cmake

```diff
@@ -50,12 +50,13 @@ elseif("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
 
         set(WARNINGS_THAT_SHOULD_BE_IGNORED # WHY it is ok to ignore        
             4702 # Got unreachable warnings from external, even though they are generallly ignored by /experimental:external
-            4710 # If the compiler decides to not inline a function, that's their decision.
-            4711 # If the compiler decides to inline a function, that's their decision.
+            4710 # If the compiler decides to not inline a function, that's their decision
+            4711 # If the compiler decides to inline a function, that's their decision
             4571 # Just a non-interesting informational warning about msvc changing behaviour in 7.1
             4267 # Conversion: Happens a lot in these samples, would complicate them too much to handle manually
             4365 # Conversion: Happens a lot in these samples, would complicate them too much to handle manually
             4388 # Signed/unsigned comparison: Happens a lot in these samples, would complicate them too much to handle manually
+            5045 # This warning is purely informational
```

Comment:
I think it would be nice if the warning error text was included (stripping away concrete type names if any). This is hard to understand without context. Like this:

```
5045 # Compiler will insert Spectre mitigation for memory load if /Qspectre switch specified
         # This warning is purely informational
```

This goes for all the warnings here

Reply:
done